doc/examples/applications/plot_fluorescence_nuclear_envelope.py

```diff
@@ -0,0 +1,214 @@
+dimensions) which shows a process of protein re-localization from the
+cytoplasmic area to the nuclear envelope. This biological application was
+first presented by Andrea Boni and collaborators in [1]_; it was used in a
+textbook by Kota Miura [2]_ as well as in other works ([3]_, [4]_).
+In other words, we port this workflow from ImageJ Macro to scikit-image.
+
+.. [1] Boni A, Politi AZ, Strnad P, Xiang W, Hossain MJ, Ellenberg J (2015)
+       "Live imaging and modeling of inner nuclear membrane targeting reveals
+       its molecular requirements in mammalian cells" J Cell Biol
+       209(5):705–720. ISSN: 0021-9525.
+       :DOI:`10.1083/jcb.201409133`
+.. [2] Miura K (2020) "Measurements of Intensity Dynamics at the Periphery of
+       the Nucleus" in: Miura K, Sladoje N (eds) Bioimage Data Analysis
+       Workflows. Learning Materials in Biosciences. Springer, Cham.
+       :DOI:`10.1007/978-3-030-22386-1_2`
+.. [3] Klemm A (2020) "ImageJ/Fiji Macro Language" NEUBIAS Academy Online
+       Course: https://www.youtube.com/watch?v=o8tfkdcd3DA
+.. [4] Vorkel D and Haase R (2020) "GPU-accelerating ImageJ Macro image
+       processing workflows using CLIJ" https://arxiv.org/abs/2008.11799
+
+"""
+
+import imageio
+import matplotlib.pyplot as plt
+import numpy as np
+import plotly.io
+import plotly.express as px
+from scipy import ndimage as ndi
+
+from skimage import (
+    filters, measure, morphology, segmentation
+)
+
+
+#####################################################################
+# We start with a single cell/nucleus to construct the workflow.
+
+dat = imageio.volread('http://cmci.embl.de/sampleimages/NPCsingleNucleus.tif')
+
+print(f'shape: {dat.shape}')
+
+#####################################################################
+# The dataset is a 2D image stack with 15 frames (time points) and 2 channels.
+
+fig = px.imshow(
+    dat,
+    facet_col=1,
+    animation_frame=0,
+    binary_string=True,
+    labels={'animation_frame': 'time point', 'facet_col': 'channel'}
+)
+plotly.io.show(fig)
+
+#####################################################################
+# To begin with, let us consider the first channel of the first image.
+
+ch0t0 = dat[0, 0, :, :]
+
+#####################################################################
+# Segment the nucleus rim
+# =======================
+
+smooth = filters.gaussian(ch0t0, sigma=2)
+
+thresh = smooth > 0.1
+
+fill = ndi.binary_fill_holes(thresh)
+
+#####################################################################
+# Although it is not strictly necessary, let us discard the other nucleus part
+# visible in the bottom right-hand corner.
+
+label = measure.label(fill)
+regions = measure.regionprops_table(label, properties=('label', 'area'))
+label_small = regions['label'][np.argmin(regions['area'])]
+label[label == label_small] = 0  # background value
+
+expand = segmentation.expand_labels(label, distance=4)
```

Comment:
In the original workflow, [we erode the mask](https://github.com/NEUBIAS/neubias-springer-book-2020/blob/master/Ch2%20Measurements%20of%20Intensity%20Dynamics%20at%20the%20Periphery%20of%20the%20Nucleus/Code/code_final.ijm#L43) and [dilate it](https://github.com/NEUBIAS/neubias-springer-book-2020/blob/master/Ch2%20Measurements%20of%20Intensity%20Dynamics%20at%20the%20Periphery%20of%20the%20Nucleus/Code/code_final.ijm#L41), which means in ImageJ to take one pixel as radius.

Reply:
Ok, fixed in 02c282357d. Does the 5ish-px-looking thickness in the final processing [step](https://link.springer.com/chapter/10.1007%2F978-3-030-22386-1_2#Fig3) only come from the rendering/printing then?